SetupIncTraceLibs.bat

```diff
@@ -18,8 +18,8 @@ call :MarkStartOfBlock "Create folders"
 call :MarkEndOfBlock "Create folders"
 
 call :MarkStartOfBlock "Retrieve dependencies"	
-	spatial package retrieve internal        trace-dynamic-x86_64-vc141_md-win32        %PINNED_CORE_SDK_VERSION% 	"%CORE_SDK_DIR%\trace_lib\trace-win32.zip"
```

Comment:
Huh wonder why we did this?

Reply:
Ahh ok this is explicitly done for this PR. Coool. wonder if we need to update the other references to the runtime version? There are some places referencing 15.1.0 -preview I believe?

Reply:
15.1.0 has fixed a compatibility problem with the legacy latency tracer (we can remove our hacks) - seemed too much for this PR though

Reply:
Does 15.1.0 have breaking api changes?

Reply:
Yeah, the renaming of the legacy trace APIs is there :tada:

Reply:
Nice - we should make sure to revert this when we actually make the fix